### PR TITLE
Wpi logging

### PR DESCRIPTION
**Pull request by asherbolotski**
replaced prints with functions that allow us to view it in advantage scope replays and we also have basic swerve logging in advantage scope

### CODE REVIEW

File: ted/src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -171,14 +177,22 @@ private void bindBrownoutActions(){
     }
   }
 
+   /**
+   * A method to init all the data logging
+   */
+  private void initializeDataLogging() {
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/RobotContainer.java]: This method should come after the next one in alphabetical order.

**Review by josieammer — CHANGES_REQUESTED** [review]
One small change requested. Otherwise LGTM.

File: ted/src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (asherbolotski) [reply ted/src/main/java/frc/robot/RobotContainer.java]: swapped

**Review by josieammer — APPROVED** [review]
LGTM!

**Pull request merged**